Tests/FileValidatorUnitTests.cs

```diff
@@ -0,0 +1,158 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml;
+using NUnit.Framework;
+using QCUtilities;
+
+namespace QCVault.Tests
+{
+    [TestFixture]
+    public class FileValidatorUnitTests
+    {
+        [TestCase(@"C:\Users\User\invalidfileName")]
+        
```

Comment:
Strip out the whitespace, butt function attributes against the function they refer to.